event.py

```diff
@@ -0,0 +1,271 @@
+                            await webhook.send(embed=embed)
+                            return
+
+    @commands.Cog.listener
+    async def on_member_unban(guild: discord.Guild, user: discord.User):
+        async for action in guild.audit_logs(action=discord.AuditLogAction.unban):
+            embed=discord.Embed(title=f"`{user.global_name}`がunBANされました",color=discord.Color.red(),timestamp=datetime.datetime.now())
+            embed.add_field(name="担当者",value=action.user.mention)
+            embed.add_field(name="ID",value=f"```{user.id}```")
+            embed.set_author(icon_url=action.user.avatar.url,name=action.user)
+            embed.set_footer(text=guild.name)
+
+            botChannel=bot.get_all_channels()
+            for channel in botChannel:
+                if channel.guild == guild:
+                    if isinstance(channel, discord.TextChannel):
+                        webhooks=await channel.webhooks()
+                        for webhook in webhooks:
+                            if webhook.name == "CalicobotLog":
+                                await webhook.send(embed=embed)
+                                return
+
+    @commands.Cog.listener
+    async def on_guild_role_create(role: discord.Role):
+        guild=role.guild
+        async for action in guild.audit_logs(action=discord.AuditLogAction.role_create):
+            embed=discord.Embed(color=discord.Color.red(),timestamp=datetime.datetime.now())
+            embed.add_field(name="",value="ロールが作成されました",inline=False)
+            embed.add_field(name="名前",value=action.target.mention)
+            embed.add_field(name="担当者",value=action.user.mention)
+            embed.set_author(icon_url=action.user.avatar.url,name=action.user.global_name)
+            embed.set_footer(text=guild.name)
+
+            botChannel=bot.get_all_channels()
+            for channel in botChannel:
+                if channel.guild == guild:
+                    if isinstance(channel, discord.TextChannel):
+                        webhooks=await channel.webhooks()
+                        for webhook in webhooks:
+                            if webhook.name == "CalicobotLog":
+                                await webhook.send(embed=embed)
+                                return
+
+    @commands.Cog.listener
+    async def on_guild_role_delete(role: discord.Role):
+        guild=role.guild
+        async for action in guild.audit_logs(action=discord.AuditLogAction.role_delete):
+            embed=discord.Embed(color=discord.Color.red(),timestamp=datetime.datetime.now())
+            embed.add_field(name="",value="ロールが削除されました",inline=False)
+            embed.add_field(name="名前",value=role.name)
+            embed.add_field(name="担当者",value=action.user.mention)
+            embed.set_author(icon_url=action.user.avatar.url,name=action.user.global_name)
+            embed.set_footer(text=guild.name)
+
+            botChannel=bot.get_all_channels()
+            for channel in botChannel:
```

Comment:
According to Pylint, variable names must follow the snake_case naming convention.
```suggestion
            bot_channel=bot.get_all_channels()
            for channel in bot_channel:
```